### DISCUSSION TITLE
Demo mode on Mia-Platform console

### DISCUSSION THREAD

**Discussion by frengooo** [mia-platform/community#53] [Ideas]
Hi guys, every time I have to show the Mia-Platform console for a demo, for the academy, or simply for a call with a different client (not the main one) I have to pay a lot of attention to avoid to show the client projects.

This is not so easy because:
- the Mia-Platform console homepage shows all the companies you can see
- the `CMD+k` shortcut shows the projects lists

We do a lot of demo and academy sessions now, for me It's very important to have this feature.
I didn't find it in the product roadmap. What do you think?

**Comment by davidebianchi** [comment]
In roadmap for the v11 there is to change how the company and projects homepage are listed. We should keep attention on this point.

**Reply by frengooo** [reply]
Do I need to create the issue on the roadmap, or comment a existing one?

**Reply by davidebianchi** [reply]
It's better to comment the already existent one, thank you!

**Reply by frengooo** [reply]
I commented it, thank you.
